test/e2e/lib/driver-helper.js

```diff
@@ -24,45 +24,63 @@ export async function highlightElement( driver, element ) {
 	}
 }
 
-export function clickWhenClickable(
-	driver,
-	selector,
-	waitOverride = null,
-	extraErrorString = null
-) {
-	const timeoutWait = waitOverride ? waitOverride : explicitWaitMS;
-	const extraErrorStringAppend = extraErrorString ? ' ' + extraErrorString : '';
+const until = {
+	...webdriver.until,
+	/**
+	 * Creates a condition that will loop until element is clickable. A clickable
+	 * element must be located and not (aria-)disabled.
+	 *
+	 * @param {By} locator The element's locator
+	 * @returns {WebElementCondition} The new condition
+	 */
+	elementIsClickable( locator ) {
```

Comment:
Neat way to logically group this as a local extension based off the `until` object 👍🏻